core/src/main/java/org/sonarsource/sonarlint/core/container/analysis/filesystem/AbsolutePathPredicate.java

```diff
@@ -42,14 +35,4 @@ public boolean apply(InputFile f) {
     return path.equals(f.absolutePath());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "absolutePath"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)